static/bidder-params/rtbhouse.json

```diff
@@ -1,8 +1,13 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
-  "title": "RTB House Adapter Params",
-  "description": "A schema which validates params accepted by the RTB House adapter",
-  "type": "object",
-  "properties": {},
-  "required": []
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "RTB House Adapter Params",
+    "description": "A schema which validates params accepted by the RTB House adapter",
+    "type": "object",
+    "properties": {
+		"publisherId": {
+			"type": "string",
+            "description": "The publisher’s ID provided by RTB House"
+        }
```

Comment:
The formatting of this document is a bit misaligned. Could you please run it through a "beautifier"? ... or align the whitespace of the `properties` object.

Reply:
converted tabs to spaces. Please have a look if it's fine? If you prefer any other linters - happy to adjust

Reply:
That works. Thank you.